### ISSUE TITLE
static flag does not works on 0.10.0 release

### ISSUE THREAD

**Issue by termoshtt** [blas-lapack-rs/openblas-src#42]
```
error: could not find native static library `openblas`, perhaps an -L flag is missing?
```

https://github.com/rust-ndarray/ndarray-linalg/pull/253

openblas-src 0.10.0 does not work with `static` flag when using as dependencies. It works if using `path`, e.g.

```toml
[dependencies.openblas-src]
version = "0.10.0"
path = "/path/to/my/local/repo"
optional = true
default-features = false
features = ["cblas"]
```

**Comment by termoshtt** [comment]
openblas-src 0.10.0 and 0.10.1 looks broken. I've yanked them.

**Comment by termoshtt** [comment]
Static linking issue is fixed in #43 and #52
